src/web_app/js/peerconnectionclient.js

```diff
@@ -29,6 +29,17 @@ var PeerConnectionClient = function(params, startTime) {
     '  constraints: \'' + JSON.stringify(params.peerConnectionConstraints) +
     '\'.');
 
+  if (typeof RTCPeerConnection.generateCertificate === 'function') {
+    RTCPeerConnection.generateCertificate({name: 'ECDSA', namedCurve: 'P-256'})
+    .then(function(cert) {
+      trace('Certificate generated successfully.');
+      params.peerConnectionConfig.certificates = [cert];
```

Comment:
Just realized this code doesn't work (AppRTC continues to use RSA certs).

You can't create the peerconnection until after the promise resolves. Might need to create the PC later, perhaps in startAsCaller/Callee.